cfme/utils/appliance/__init__.py

```diff
@@ -251,12 +251,12 @@ def __init__(
             if not (address.scheme and address.netloc):
                 # Use .path (w.x.y.z ip format)
                 self.address = address.path
-                self.scheme = "https"
+                self._scheme = "https"
                 self._url = "https://{}/".format(address.path)
             else:
                 # schema://w.x.y.z/ format
                 self.address = address.netloc
-                self.scheme = address.scheme
+                self._scheme = address.scheme
```

Comment:
**REQUIRED** based on the __init__ method it seems that `scheme` is always set, so we should drop the property and just have the attribute

additionally at second looks the change is a breaking api change since appliance.scheme turns from writable to unwritable

from my pov the easiest way to sort that out is to undo the rename in `__init__` and dropping the property altogether

Reply:
you're right.  I was doing these changes late night and missed that. Thank you